requirements.txt

```diff
@@ -1,4 +1,4 @@
 backoff
 ruamel.yaml
 auth0-python
-git+https://github.com/GeorgianaElena/jupyterhub_client.git@678d4733482b1196fcff3c05bd1fdb134777a475
\ No newline at end of file
+git+https://github.com/GeorgianaElena/jupyterhub_client.git@7aa8e4c8ee2da9b4d98065e9b19b766ab8f66be8
```

Comment:
Going onwards, this is supposed to be https://github.com/Quansight/jhub-client, is that correct?

Reply:
Yep! We just need this one PR https://github.com/Quansight/jhub-client/pull/5 merged before we can use the real thing <3

Reply:
Ah, nice work on this @GeorgianaElena :tada:!

Reply:
Yeup sorry should have given a heads up that the repository changed! Luckily github does redirects for awhile.